k8s/namespaces/ccd/ccd-data-store-api/perftest.yaml

```diff
@@ -21,7 +21,7 @@ spec:
         CCD_S2S_AUTHORISED_SERVICES_CASE_USER_ROLES: aac_manage_case_assignment,fpl_case_service,iac,finrem_case_orchestration,civil_service,divorce_frontend,nfdiv_case_api,civil_service,adoption_cos_api,adoption_web,prl_cos_api
         DATA_STORE_DB_HOST: ccd-data-store-api-postgres-db-v11-perftest.postgres.database.azure.com
         DATA_STORE_DB_USERNAME: ccd@ccd-data-store-api-postgres-db-v11-perftest
-        DATA_STORE_S2S_AUTHORISED_SERVICES: ccd_data,ccd_gw,ccd_ps,probate_backend,divorce_ccd_submission,sscs,sscs_bulkscan,cmc,cmc_claim_store,cmc_claim_external_api,jui_webapp,pui_webapp,bulk_scan_orchestrator,fpl_case_service,iac,finrem_ccd_data_migrator,finrem_case_orchestration,employment_tribunals,ethos_repl_service,ccpay_bubble,ctsc_work_allocation,em_ccd_orchestrator,xui_webapp,bulk_scan_payment_processor,pcq_consolidation_service,em_npa_app,ecm_consumer,aac_manage_case_assignment,am_role_assignment_service,nfdiv_cms,nfdiv_cos,divorce_frontend,wa_task_configuration_api,wa_task_management_api,wa_workflow_api,em_hrs_api,nfdiv_case_api,civil_service,wa_task_monitor,ccd_case_document_am_api,adoption_cos_api,adoption_web,prl_cos_api,et_sya_api,ds_ui,prl_citizen_frontend,hmc_cft_hearing_service,prl_cos_api,ccd_next_hearing_date_updater
+        DATA_STORE_S2S_AUTHORISED_SERVICES: ccd_data,ccd_gw,ccd_ps,probate_backend,divorce_ccd_submission,sscs,sscs_bulkscan,cmc,cmc_claim_store,cmc_claim_external_api,jui_webapp,pui_webapp,bulk_scan_orchestrator,fpl_case_service,iac,finrem_ccd_data_migrator,finrem_case_orchestration,employment_tribunals,ethos_repl_service,ccpay_bubble,ctsc_work_allocation,em_ccd_orchestrator,xui_webapp,bulk_scan_payment_processor,pcq_consolidation_service,em_npa_app,ecm_consumer,aac_manage_case_assignment,am_role_assignment_service,nfdiv_cms,nfdiv_cos,divorce_frontend,wa_task_configuration_api,wa_task_management_api,wa_workflow_api,em_hrs_api,nfdiv_case_api,civil_service,wa_task_monitor,ccd_case_document_am_api,adoption_cos_api,adoption_web,prl_cos_api,et_sya_api,ds_ui,prl_citizen_frontend,hmc_cft_hearing_service,prl_cos_api,ccd_next_hearing_date_updater,sptribs_dss_backend,civil_general_applications
```

Comment:
This line shouldn't be changed as part of any of the Redis changes as it's unrelated